design/ceph/object/zone.md

```diff
@@ -134,6 +143,8 @@ The following variables can be configured in the ceph-object-zone resource.
 
 - `customEndpoints`:  Specify the endpoint(s) that will accept multisite replication traffic for this zone. You may include the port in the definition if necessary. For example: "https://my-object-store.my-domain.net:443".
 
+- `preservePoolsOnDelete`: If it is set to 'true' the pools used to support the zone will remain when the ceph object zone CR will be deleted. This is a security measure to avoid accidental loss of data. It is set to 'true' by default. If not specified is also deemed as 'true'.
```

Comment:
Should the default be `false` if it's not a master zone? It's not critical to preserve the pools in that case, right?

Reply:
@travisn: That's true. Are suggesting the following:
If user sets the value then directly use that value, if not have a helper function to determine the value depending on the master or peer zone than using the inbuild Kubernetes default value.
One another reason to set default value as `true` to preseve the old behaviour of pools not getting removed from the backend even if Zone CR is removed. @alimaredia please share your opinion